### PR TITLE
Card12/normal admin

### PR DESCRIPTION
**Pull request by dscherer21**

### CODE REVIEW

**Comment by vhoof** [comment]
You'll need to merge qa and fix merge conflicts for sure.

File: src/ProfilesPage/ProfilePage.js [hunk 1]
```diff
@@ -118,7 +129,10 @@ class ProfilePage extends Component {
 
   onDelete() {
     const id = this.props.match.params.id || sessionStorage.getItem("userId");
-    this.props.deleteUser(id);
+    alert(id);
+    alert('Button Clicked!')
+    {/*this.props.deleteUser(id);*/}
```
Comment (vhoof) [inline src/ProfilesPage/ProfilePage.js]: What's going on here?

File: src/ProfilesPage/ProfilePage.js [hunk 2]
```diff
@@ -75,8 +84,10 @@ class ProfilePage extends Component {
   }
 
   renderEditUserButton() {
+    console.log(this.state.editUserEnable);
```
Comment (vhoof) [inline src/ProfilesPage/ProfilePage.js]: Can remove this console log.

File: src/ProfilesPage/ProfilePage.js [hunk 3]
```diff
@@ -22,26 +22,35 @@ import UserInfo from "./components/UserInfo";
...
+    fetchUserInfo(this.props.match.params.id);
+    fetchUserFeeds(this.props.match.params.id);
+    fetchUserConnections(this.props.match.params.id);
+    fetchConnectionStatus({ sender_id, receiver_id });
+    fetchNotifications(sender_id);
     this.setState(() => {
       return { suspendedState: this.props.userInfo.suspended };
     });
-    this.setState({ editUserEnable: false });
+    /*this.setState({ editUserEnable: false });*/
```
Comment (vhoof) [inline src/ProfilesPage/ProfilePage.js]: Can just remove the commented stuff.

File: src/ProfilesPage/ProfilePage.js [hunk 4]
```diff
@@ -75,8 +84,10 @@ class ProfilePage extends Component {
   }
 
   renderEditUserButton() {
+    console.log(this.state.editUserEnable);
     return (
       <button className="btn btn-info" onClick={this.handleEditButtonClick.bind(this)}>
+        {/*if this.state.editUserEnable === false render "back" else render "Edit"*/}
```
Comment (vhoof) [inline src/ProfilesPage/ProfilePage.js]: Can just remove the commented stuff.

File: src/ProfilesPage/ProfilePage.js [hunk 5]
```diff
@@ -162,13 +176,19 @@ class ProfilePage extends Component {
...
-        <button className="btn btn-danger" onClick={this.onDelete.bind(this)}>
-          Delete User
-        </button>
-      </a>
-    );
+    const superuser = this.props.userInfo.superuser;
+    
+    console.log(superuser);
+    {/*if superuser's value is false render deleleteUser button.*/}
+    if(superuser === false) {
```
Comment (vhoof) [inline src/ProfilesPage/ProfilePage.js]: Could just use `if (!this.props.userInfo.superuser)` in place of creating the local `const`. And delete the console log and comment on 181-182.

File: src/ProfilesPage/ProfilePage.js [hunk 6]
```diff
@@ -162,13 +176,19 @@ class ProfilePage extends Component {
...
+      return (
+        <a href="/message">
+          <button className="btn btn-danger" onClick={this.onDelete.bind(this)}>
+            Delete User
+          </button>
+        </a>
```
Comment (vhoof) [inline src/ProfilesPage/ProfilePage.js]: I'm having trouble understanding the `button` wrapped in an `a`?

File: src/ProfilesPage/components/UserInfo.js [hunk 7]
```diff
@@ -23,7 +23,7 @@ const UserInfo = (props) => {
   if (props.userInfo) {
 
 
-    const { id, suspended, firstName, lastName, bio, block_connection_requests, admin } = props.userInfo;
+    const { id, suspended, firstName, lastName, bio, block_connection_requests, admin, superuser } = props.userInfo;
```
Comment (vhoof) [inline src/ProfilesPage/components/UserInfo.js]: I don't think you need this `superuser` here. Actually, based on what's used below, you should only need `firstName`, `lastName`, and `bio`.

File: src/common/Navbar.js [hunk 8]
```diff
@@ -72,9 +72,10 @@ class Navbar extends Component {
     this.props.history.push(`/profile/${item.value}`);
   }
 
-  componentWillReceiveProps(newProps) {
-    const { userInfo } = newProps;
-  }
+  //Not sure what this line is supposed to be doing?
+  //componentWillReceiveProps(newProps) {
+    //const { userInfo } = newProps;
+  //}
```
Comment (vhoof) [inline src/common/Navbar.js]: I think? we can just delete this.

File: src/ProfilesPage/ProfilePage.js (hunk 1 above)
Comment (dscherer21) [reply src/ProfilesPage/ProfilePage.js]: done

File: src/ProfilesPage/ProfilePage.js (hunk 2 above)
Comment (dscherer21) [reply src/ProfilesPage/ProfilePage.js]: done

File: src/ProfilesPage/ProfilePage.js (hunk 3 above)
Comment (dscherer21) [reply src/ProfilesPage/ProfilePage.js]: done

File: src/ProfilesPage/ProfilePage.js (hunk 4 above)
Comment (dscherer21) [reply src/ProfilesPage/ProfilePage.js]: this seems more of a note explaining what the functioning is doing

File: src/ProfilesPage/ProfilePage.js (hunk 5 above)
Comment (dscherer21) [reply src/ProfilesPage/ProfilePage.js]: done

File: src/ProfilesPage/ProfilePage.js (hunk 6 above)
Comment (dscherer21) [reply src/ProfilesPage/ProfilePage.js]: not sure why that was like that but it is fixed.

File: src/ProfilesPage/components/UserInfo.js (hunk 7 above)
Comment (dscherer21) [reply src/ProfilesPage/components/UserInfo.js]: done

File: src/common/Navbar.js (hunk 8 above)
Comment (dscherer21) [reply src/common/Navbar.js]: removed

**Comment by dscherer21** [comment]
@vhoof  @mkirlin  should be ready for merge

**Comment by vhoof** [comment]
Now there are merge conflicts again. 😬

**Pull request merged**